draft-ietf-tls-tls13.md

```diff
@@ -1895,10 +1895,7 @@ appears it MUST abort the handshake with an "illegal_parameter" alert.
 |:-----------------------------------------|------------:|
 | server_name [RFC6066]                    |      CH, EE |
 | max_fragment_length [RFC6066]            |      CH, EE |
-| client_certificate_url [RFC6066]         |      CH, EE |
```

Comment:
We didn't discuss the client_certificate_url extension during the Chicago IETF meeting but I am wondering whether the considerations for it are somewhat similar than for the cached info extension. In an IoT context you most likely want to use TLS/DTLS for mutual authentication and to avoid sending the client certificate URI extension allows the client to avoid transmitting the full certificate. While it is not as efficient as cached info does for the server certificate it is still a significant improvement over sending the full certificate.